Documentation/Links/LinkActions.md

```diff
@@ -86,6 +86,7 @@ After these configurations are set, when the user clicks on the link, it will op
 
 Use this to open Custom Views in the Azure Portal. Verify all of the configuration and settings. Incorrect values will cause errors in the portal, or fail to open the views correctly. There are two ways to configure the settings, via the `Form` or via `URL`.
 
+>**_NOTE:_** Views with a menu cannot be opened in a Context Blade.

```

Comment:
what happens if you do it?  you might want to explain what the symptoms are, like you'll get a specific error, etc.  that way if anyone ever searches the repo for it it might get found?